src/clj/nnts2/db.clj

```diff
@@ -1,7 +1,7 @@
 (ns nnts2.db
   (:require [ragtime.jdbc :as jdbc]
             [ragtime.repl :as repl]
-            [nnts2.config :refer [db-spec]]))
+            [nnts2.config :as config :refer [db-spec]]))
```

Comment:
We can have either config or refer to db-spec. In making it constant throughout the project, I would say use :as config and then config/db-spec